jmxtrans-docker-test/pom.xml

```diff
@@ -0,0 +1,426 @@
+                                    <run>
+                                        <cmd>
+                                            <exec>
+                                                <arg>/bin/bash</arg>
+                                                <arg>-c</arg>
+                                                <arg>${deb.run.cmd}</arg>
+                                            </exec>
+                                        </cmd>
+                                        <wait>
+                                            <log>${verification.message}</log>
+                                            <time>${verification.wait.time}</time>
+                                        </wait>
+                                    </run>
+                                </image>
+
+                                <image>
+                                    <name>debian-7.9/${project.artifactId}:${project.version}</name>
+                                    <build>
+                                        <assembly>
+                                            <descriptor>assembly-docker.xml</descriptor>
+                                            <basedir>/</basedir>
+                                        </assembly>
+
+                                        <from>debian:7.9</from>
+                                        <runCmds>
+                                            <run>apt-get update</run>
+                                            <run>apt-get -y install openjdk-7-jre</run>
+                                        </runCmds>
+                                    </build>
+                                    <run>
+                                        <cmd>
+                                            <exec>
+                                                <arg>/bin/bash</arg>
+                                                <arg>-c</arg>
+                                                <arg>${deb.run.cmd}</arg>
+                                            </exec>
+                                        </cmd>
+                                        <wait>
+                                            <log>${verification.message}</log>
+                                            <time>${verification.wait.time}</time>
+                                        </wait>
+                                    </run>
+                                </image>
+                            </images>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>rpm</id>
+
+            <properties>
+                <rpm.install.artifact>org.jmxtrans:jmxtrans:rpm</rpm.install.artifact>
+                <rpm.install.file>jmxtrans.rpm</rpm.install.file>
+                <rpm.run.cmd>rpm -ivh --nosignature --nodeps /maven/${rpm.install.file}; /etc/init.d/jmxtrans start; /maven/verify.sh</rpm.run.cmd>
```

Comment:
I would expect jmxtrans to start running when the package is installed. So `/etc/init.d/jmxtrans start` should probably not bee required.